docs/release-playbook.md

```diff
@@ -122,16 +122,13 @@ Create candidates with the release.sh script.
         automatically run. Make sure that it passes.
 
 1.  When it all looks good, go back to the job in the release pipeline, click
-    "Unblock step" for the deployment step. This will upload the release
-    candidate binaries to GitHub, https://releases.bazel.build and our apt-get
-    repository.
+    "Unblock step" for the deployment step. 
+    
+    *   This will upload the release candidate binaries to GitHub and our 
+        apt-get repository. The github link is probably of the form:
+        https://releases.bazel.build/0.25.0/rc1/index.html
 
-    *   If you don't have the permission, ask one of the Buildkite org
-        admins to add you to the
-        [bazel-sheriffs](https://buildkite.com/bazel?team=bazel-sheriffs)
```

Comment:
Please keep this paragraph, but change the group to https://buildkite.com/organizations/bazel-trusted/teams/release-managers/members.

Membership is still required (you're already a member of that group).

Reply:
Context: we moved the release pipeline to a separate Buildkite org (bazel-trusted), which required the creation of a new group.

Reply:
@dkelmer Friendly ping :)

Reply:
Sorry, this totally fell off my radar. Addressed the changes!